### PR TITLE
add retrolambda support again

### PR DESCRIPTION
**Pull request by Piasy**

### CODE REVIEW

File: buildSrc/src/main/groovy/com/github/piasy/okbuck/config/RetroLambdaShFile.groovy [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    public RetroLambdaShFile(String classpath, String buildTarget, String retroLambdaJarFileName) {
+        mClasspath = classpath
+        mBuildTarget = buildTarget
+        mRetroLambdaJarFileName = retroLambdaJarFileName
+    }
+
+    @Override
+    public final void print(PrintStream printer) {
+        printer.println("#!/bin/bash")
+        String androidJarPath = System.getenv("ANDROID_HOME") + File.separator + "platforms" +
```
Comment (kageiit) [inline buildSrc/src/main/groovy/com/github/piasy/okbuck/config/RetroLambdaShFile.groovy]: this assumption is not always valid. not everyone has ANDROID_HOME set

Comment (Piasy) [inline buildSrc/src/main/groovy/com/github/piasy/okbuck/config/RetroLambdaShFile.groovy]: it's required by buck. buck installation guide required user to set this variable.

**Comment by kageiit** [comment]
I believe we can clean this up a lot. Ill commit on top of this branch. thanks for the initial idea!

File: buildSrc/src/main/groovy/com/github/piasy/okbuck/config/RetroLambdaShFile.groovy (hunk 1 above)
Comment (kageiit) [inline buildSrc/src/main/groovy/com/github/piasy/okbuck/config/RetroLambdaShFile.groovy]: I know it needs it. But this is not the right way to do it as user may not have ANDROID_HOME environment variable set always.

I will post the more reliable method on top of this branch.

**Pull request merged**